### ISSUE TITLE
Emotecolor

### ISSUE THREAD

**Issue by orishi** [Sphereserver/Source-X#358]
Please, if already not included, add <emotecolor> property to characters, as they did on 56d.
https://github.com/Sphereserver/Source/commit/5b29e68c2b58694c86bc181f62e9926606d0d2c7

**Comment by meaowR** [comment]
would be great cause i want to change some emotes to green or yellow too :)

**Comment by orishi** [comment]
I'm afraid you didn't understand what I needed.
I need to send SAYU in the same color as the client/player has set for himself. I need to read client's emote speech color settings in client options.

As on 56d:
https://github.com/Sphereserver/Source/commit/5b29e68c2b58694c86bc181f62e9926606d0d2c7

**Comment by xwerswoodx** [comment]
Firstly, the code you send is a color override for emotes. So I will do same thing but in different way because both versions has different codes.

Secondly, we already have SpeechColorOverride for SAY overrides, so could you check it? Because it's already exist. If it's not working for SAYU, I can check.

**Comment by orishi** [comment]
Well, maybe it's just that english is not my native language, so let me rephrase.

First of all, this is already implemented and will display "some text" in the same color as the client/player has set in the client.exe options:
`sayua <speechcolor>,0,0,1 some text`

I need the same thing but for emote color. Or did I not understand your code and it's solved?

**Comment by xwerswoodx** [comment]
Hmm, I guess I get you know, I implemented another EMOTECOLOR setting to get emote color from client-side. I will return after I made some test.

**Comment by orishi** [comment]
Great we understand each other now, let me know, thanks.

**Comment by orishi** [comment]
Works great, thanks.
